impl/client/src/main/javascript/web/dojo/pentaho/common/RowLimitControl.js

```diff
@@ -52,6 +52,9 @@ define(["dojo/_base/declare", "dijit/form/Select", "dijit/form/ValidationTextBox
               this.inherited(arguments);
               on(this.rowLimitRestrictions, 'change', lang.hitch(this, '_onSelect'));
               on(this.rowsNumberInput, 'keydown, focusout', lang.hitch(this, '_onRowLimitSubmit'));
+              on(this.rowsNumberInput, 'mousedown', function(e){ 
+                e.preventDefault(); 
```

Comment:
I think will be better to check here disabled or not the input and use `e.preventDefault();` only if the input is disabled.